content/post/2019-11-16-robot-framework-for-network-testing.md

```diff
@@ -0,0 +1,20 @@
+---
+date: 2018-11-16T00:00:00-00:00
+description: "Automated testing made easy and efficient with Robot framework"
+featured_image: "/images/nrelabs.png"
+tags: [verification, testing]
+title: "Using Robot Framework for Automated Testing"
+show_reading_time: true
+authors:
+  name: Saim Khan
```

Comment:
I'll review the rest later, but for now, I'll mention that the current template doesn't support multiple authors. however, this is an easy fix, and I'll make the necessary changes. In the meantime, please convert this to a list of dicts - in the current format, the key `Saim Khan` would get overwritten.

I.e.:

```
- name: Saim Khan
  link: https://github.com/saimkhan92
- name: Lakshmi Rajan
  link: https://github.com/lara29
```

Reply:
Thanks Matt, I've changed the authors to list of dicts and committed